src/apple/apple-wizard.ts

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable max-lines */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import clack from '@clack/prompts';
+import chalk from 'chalk';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as templates from './templates';
+import * as xcManager from './xcode-manager';
+import * as codeTools from './code-tools';
+import * as bash from '../utils/bash';
+import { WizardOptions } from '../utils/types';
+
+const xcode = require('xcode');
+/* eslint-enable @typescript-eslint/no-unused-vars */
+
+import {
+  askForProjectSelection,
+  askForSelfHosted,
+  askForWizardLogin,
+  askToInstallSentryCLI,
+  SentryProjectData,
+  printWelcome,
+} from '../utils/clack-utils';
+
+export async function runAppleWizard(
+  options: WizardOptions,
+): Promise<void> {
+  printWelcome({
+    wizardName: 'Sentry Apple Wizard',
+    promoCode: options.promoCode,
+  });
+
+
+  if (!bash.hasSentryCLI()) {
+    if (!(await askToInstallSentryCLI())) {
+      clack.log.warn("Without Sentry-cli, you won't be able to upload debug symbols to Sentry. You can install it later by following the instructions at https://docs.sentry.io/cli/");
+    } else {
+      await bash.installSentryCLI();
+    }
+  }
+
+  const projectDir = process.cwd();
+  const xcodeProjFile = findFilesWithExtension(projectDir, ".xcodeproj")[0];
+
+  if (!xcodeProjFile) {
+    clack.log.error('No xcode project found. Please run this command from the root of your project.');
+    return;
```

Comment:
One thing I'm also realizing here is that we shouldn't return from the wizard flow without first aborting. Calling `await abort` from `clack-utils.ts` will make sure, we exit with an error exit code and (if telemetry data is collected) we flush out pending Sentry events before exiting. 

I also did this incorrectly in the SvelteKit wizard. Will open an issue to fix there.